script/utils/util.py

```diff
@@ -109,14 +109,14 @@ def get_dst_path(base_dir, target_platform, debug):
 
 
 def push_to_android(src, dst):
-    run_cmd(f'adb -d push {src} {ANDROID_BASE}{dst}')
+    run_cmd(f'adb push {src} {ANDROID_BASE}{dst}')
```

Comment:
Why should we delete these flags? "-d" is necessary to specify the physical device if adb detects it along with android simulators. 

It also seems irrelevant to the PR.

Reply:
I deleted the `-d` option because it **doesn't support wireless connection** with adb. It is for the usb connection.
Do you think using `-d` option is more necessary?

Reply:
I see, maybe we should add additional option dedicated for any flags at the end of the ArgParse.

Reply:
I'll separately handle this issue in another PR.